plugins/out_stackdriver/stackdriver.c

```diff
@@ -34,6 +34,8 @@
 #include <fluent-bit/flb_log_event_decoder.h>
 #include <fluent-bit/flb_gzip.h>
 
+#include <cmetrics/cmt_histogram.h>
```

Comment:
Since this needs to be included in `stackdriver.c` and `stackdriver_conf.c`, let's include it in `stackdriver.h` since that's already included in both spots.

Reply:
Done.